v4v/dkms.conf

```diff
@@ -1,7 +1,7 @@
 PACKAGE_NAME="v4v"
 PACKAGE_VERSION="1.0"
+MAKE[0]="make KVER=$kernelver"
```

Comment:
Why is this change made?  v4v/dkms.conf also changes in two of the following patches.

Reply:
TL;DR: Because I did not rebase my changes correctly it seems ;).

I initially changed the {{Makefile}} and {{dkms.conf}} in a guest I had and saved the files, then integrated it in OE. Since the [doc](https://www.kernel.org/doc/Documentation/kbuild/modules.txt) used {{KDIR}} I had {{KVER}} before remembering that {{module.bbclass}} uses {{KERNEL_VERSION}}.

Reply:
Ah, ok.  We've all been there with rebasing problems :)